docs/input.md

```diff
@@ -90,20 +90,199 @@ def setup(): # Only drawn once, so just using setup()
 
-```python
+```python hl_lines="5"
 %%ignite
 
 def setup(): # Only drawn once, so just using setup()
     size(200, 200)
-    radius = int(input("What radius should the circle be?: ")) # cast the string to an integer
-    fill_circle(100, 100 radius)
+    diameter = int(input("What diameter should the circle be?: ")) # cast the string to an integer
+    fill_circle(100, 100 diameter)
 ```
 
 which results in:
 
-![Ask circle radius](img/circle_radius.png)
+![Ask circle diameter](img/circle_radius.png){: loading=lazy }
+
+### Check for key presses
+
+Spark features multiple ways to check for user key presses:
+
+| Name                            | Type                  | Description                                                  |
+|---------------------------------|-----------------------|--------------------------------------------------------------|
+| [key](#key)                     | Variable              | A string of what key was last pressed                        |
+| [keys_held()](#keys_held)       | Function              | A function that returns True if the provided key is held     |
+| [key_pressed()](#key_pressed)   | User defined function | A definable function that activates when any key is pressed  | 
+| [key_released()](#key_released) | User defined function | A definable function that activates when any key is released |
+| [key_repeated()](#key_repeated) | User defined function | A definable function that activates when any key is held     |
+
+#### key
+
+A string that is the last key pressed
+
+##### Notes
+- This variable **does not clear**, meaning if a key is pressed it will remain the value until a **new key** is pressed
+- This variable will be the uppercase letter if ++shift++ + a letter is pressed i.e. ++shift++ + ++a++ would be `#!python key == "A"`
+- This variable will be a symbol if ++shift++ + a number is pressed i.e. ++shift++ + ++3++ would be `#!python key == "$"`
+
+
+##### Special Keys
+
+| Key           | Value                          |
+|---------------|--------------------------------|
+| ++up++        | `#!python key == "ArrowUp"`    |
+| ++down++      | `#!python key == "ArrowDown"`  |
+| ++left++      | `#!python key == "ArrowLeft"`  |
+| ++right++     | `#!python key == "ArrowRight"` |
+| ++tab++       | `#!python key == "Tab"`        |
+| ++ctrl++      | `#!python key == "Control"`    |
+| ++alt++       | `#!python key == "Alt"`        |
+| ++esc++       | `#!python key == "Escape"`     |
+| ++win++       | `#!python key == "Meta"`       |
+| ++backspace++ | `#!python key == "Backspace"` |
+
+**Example(s):**
+
+*Print out each key as they are pressed (I pressed **just** the ++a++ key in this example)*
+
+
+```python hl_lines="8"
+%%ignite
+def setup():
+    size(500,500)
+
+def draw():
+    background(255)
+    text_size(32)
+    text(key, 250, 200)
+```
+
+Results in:
```

Comment:
On my local version, when this compiles the "Results in: " shows below the image, I don't know if there's a way to fix this, or if we care

Reply:
Ah, it looks like it needs one newline of padding or the lazyloading causes the dom elements to be flipped if the page isn't on that section. I have resolved this in the most recent commit.